exampleSite/content/posts/theme-documentation-basics.md

```diff
@@ -42,266 +41,316 @@ git submodule add https://github.com/526avijitgupta/gokarna.git themes/gokarna
 
 ```bash
 hugo
 ```
 
-A `public` folder will be generated, containing all static content and assets for your website. It can now be deployed on any web server.
+A `public` folder will be generated, containing all static content and assets for your website. It can be hosted on any web server.
 
-The website can be automatically published and hosted with [Netlify](https://www.netlify.com/), [AWS Amplify](https://gohugo.io/hosting-and-deployment/hosting-on-aws-amplify/), [Github pages](https://gohugo.io/hosting-and-deployment/hosting-on-github/), [Render](https://gohugo.io/hosting-and-deployment/hosting-on-render/) and more...
+The website can be automatically published and hosted with [Netlify](https://www.netlify.com/), [AWS Amplify](https://gohugo.io/hosting-and-deployment/hosting-on-aws-amplify/), [GitHub Pages](https://gohugo.io/hosting-and-deployment/hosting-on-github/), [Render](https://gohugo.io/hosting-and-deployment/hosting-on-render/), and more...
 
 ## Configuration
 
-In addition to [Hugo global configuration](https://gohugo.io/overview/configuration/) and [menu configuration](#basic-configuration), **gokarna** lets you define the following parameters in your site configuration (here is a `config.toml`, whose values are default).
+In addition to [Hugo global configuration](https://gohugo.io/overview/configuration/) and [menu configuration](#basic-configuration), Gokarna lets you define the following [parameters](https://gohugo.io/methods/site/params/) in your site configuration.
+
+See this sample `config.toml`, which uses Gokarna's default values:
 
 ```toml
 [params]
-  # URL for the avatar on homepage
+  # Resource URL for the avatar (seen on the home page and header)
   avatarURL = ""
 
-  # Choose one of size-xs, size-s, size-m, size-l & size-xl. (Default: size-m)
-  avatarSize = ""
+  # Describe the avatar for screen readers
+  avatarAltText = ""
 
-  # Description to display on homepage
-  description = "Sky above, sand below & peace within"
+  # Set the avatar's size. Valid values include:
+  # size-xs, size-s, size-m, size-l & size-xl
+  avatarSize = "size-m"
 
-  # Accent color is displayed when you hover over <a> tags
+  # Choose the color shown when hyperlinks are hovered over
   accentColor = "#FF4D4D"
 
-  # Display "back to top" button on posts and pages
-  ShowBackToTopButton = true
-
-  # You can use this to inject any HTML in the <head> tag.
-  # Ideal usecase for this is to import custom js/css or add your analytics snippet
+  # Inject arbitrary HTML via the <head> tag
+  # Best used for importing custom JavaScript, CSS, or adding an analytics snippet
   customHeadHTML = ""
 
-  # Keywords relevant for SEO
-  metaKeywords = ["blog", "gokarna", "hugo"]
+  # Configure how post dates are displayed
+  # dateFormat must be set if lastmod is declared in front matter, or
+  # enableGitInfo is true
+  dateFormat = "January 2, 2006"
+  
+  # Meta description shown on the home page
+  description = ""
```

Comment:
Let's keep using "Sky above, sand below & peace within" instead of keeping the `description` empty. Serves as an easy to follow reference for users.

Reply:
No problem, I separated the exampleSite config and Gokarna defaults because they were conflated to be the same thing. I'll add a line to express that the two are used in tandem